app/models/agents/webhook_agent.rb

```diff
@@ -20,6 +20,9 @@ class WebhookAgent < Agent
         * `payload_path` - JSONPath of the attribute in the POST body to be
           used as the Event payload.  If `payload_path` points to an array,
           Events will be created for each element.
+        * `verbs` - Comma-separated list of http verbs your agent will accept.
```

Comment:
Case-in-point:  Why is this agent blocking on verbs.. at all?

This agent originally blocked to only POST request.  What use-case was met by programming this block?  And look how it persists... I asked if it would be ok if GET could be allowed, which would be a simple change.  But I'm told that instead, we should allow a set of http verbs.  So I do that, and allow both comma and semicolon delimiters... and I'm told we must pick one.

Think of all this code and talking that could have been avoided if that one arbitrary block wasn't programmed in the fist place.

I don't mean to poke at anybody; I've done this sort of stuff a lot in my career.  Gosh, I still do it.  We're all guilty of it. This should be a lesson on how we should be checking ourselves - noting when the software is driven by actual business need, and when it's driven by a arbitrary decisions from a few programmers.